adapters/pubmatic/pubmatic.go

```diff
@@ -569,8 +569,18 @@ func getBidType(bidExt json.RawMessage) openrtb_ext.BidType {
 		if err == nil {
 			err = json.Unmarshal(extbyte, &bidExtMap)
 			if err == nil && bidExtMap[bidTypeExtKey] != nil {
-				bidTypeStr := bidExtMap[bidTypeExtKey].(string)
-				bidType = openrtb_ext.BidType(bidTypeStr)
+				bidTypeVal := int(bidExtMap[bidTypeExtKey].(float64))
```

Comment:
Converting to a float and then an int mean that you are safe on the conversions if someone entered a float, as opposed to casting it directly to an int?